book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] please kindly ensure that Getters and Setters for @rentals should be present
eg `attr_accessor :title, :author, :rentals`